mtp_transaction_uploader/upload.py

```diff
@@ -225,6 +230,28 @@ def extract_sender_information(record):
     )
 
 
+def update_new_balance(transactions, date):
+    conn = get_authenticated_connection()
+    response = conn.balances.get(limit=1, date__lt=date.isoformat())
+    if response.get('results'):
+        balance = response['results'][0]['closing_balance']
+    else:
+        balance = 0
+
+    for t in transactions:
+        if t['category'] == 'credit':
+            balance += t['amount']
+        elif t['category'] == 'debit':
+            balance -= t['amount']
+
+    try:
+        conn.balances.post({'date': date.isoformat(),
+                            'closing_balance': balance})
+    except HttpClientError:
+        # this is probably fine, just means that balance exists
```

Comment:
shouldn't we log this though? also, would it be a good idea to get the api to accept an overwrite for a date? or would that screw up reconciliation?